doc/man3/SSL_CTX_set_verify.pod

```diff
@@ -109,8 +109,9 @@ may be different.
 
 SSL_CTX_set_verify_depth() and SSL_set_verify_depth() set the limit up
 to which depth certificates in a chain are used during the verification
-procedure. If the certificate chain is longer than allowed, the certificates
-above the limit are ignored. Error messages are generated as if these
+procedure. Neither end-entity nor the trust-anchor certificates count against
+B<depth>. If the certificate chain is longer than B<depth+2>, the certificates
+above the limit are hence ignored. Error messages are generated as if these
```

Comment:
will do that. for the time being, i've removed the piece about error messages. I'll rebase after we've reached a more stable version. Is that ok?